apps/web/src/lang/fr.json

```diff
@@ -1335,6 +1335,10 @@
     "defaultMessage": "Erreur : échec de la mise à jour de l'utilisateur",
     "description": "Message displayed to user after user fails to get updated."
   },
+  "5FK/4R": {
+    "defaultMessage": "Échéancier de carrière",
```

Comment:
This translation is inconsistent with how **Career timeline** has been translated before in our codebase.

`2KM4iz`, `TUfJUD`, `sdvCNP` = _Parcours professionnel_